frb_dart/lib/src/basic.dart

```diff
@@ -25,6 +33,21 @@ abstract class FlutterRustBridgeBase<T extends FlutterRustBridgeWireBase> {
   @protected
   final T inner;
 
+  late final _dropPort = _initDropPort();
+  NativePortType get dropPort => _dropPort.sendPort.nativePort;
+
+  ReceivePort _initDropPort() {
+    var port = broadcastPort(_DropIdPortGenerator.instance.create());
+    port.listen((message) {}).onData(((response) {
```

Comment:
What about

`port.listen((message) { ... the body });`

Reply:
I will double check, but in my opinion it does not work (for some unknown reason)

Reply:
Hmm, it seems to work, but it may be unstable. let it be `listen`